src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/Services/WorkflowTrimmingService.cs

```diff
@@ -46,7 +46,7 @@ public async Task<int> TrimWorkflowInstancesAsync(TimeSpan retentionPeriod, int
             return 0;
         }
 
-        var statuses = settings.Statuses.Select(x => (int)x).ToArray();
+        var statuses = settings.Statuses.ToArray();
```

Comment:
You don't need this mutation altogether since it's already an array. So remove .ToArray()